gammapy/estimators/points/core.py

```diff
@@ -139,14 +142,20 @@ def read(
             kwargs.setdefault("format", "ascii.ecsv")
             table = Table.read(filename, **kwargs)
 
+        try:
+            gti = GTI.read(filename)
+        except (TypeError, AstropyDeprecationWarning):
```

Comment:
Why catching the `AstropyDeprecationWarning`?
This might be problematic because we need to see those warnings to remove the deprecated functionality.

Reply:
If the required HDU is not present, ie, GTI, then it tries to read the first HDU. This leads to a warning and a OSError much deeper down. Catching the OSError did not work.